src/receive_imf.rs

```diff
@@ -1484,7 +1484,7 @@ async fn add_parts(
             )
             .await?;
             let is_fresh_reaction = !seen && !fetching_existing_messages;
-            if mime_parser.incoming && is_fresh_reaction {
+            if mime_parser.incoming && is_fresh_reaction && !reaction.is_empty() {
```

Comment:
Why not emit an event when a reaction is removed? The user will continue thinking there's a reaction that doesn't exist anymore. But that would require a translation for "unreacted" or smth like this, have no better idea

Reply:
yes, maybe. not sure if it is worth going that path, seems a cornercase. i see these notification more as "sth. was happening", not that the user tracks all notifications and then knows the final state :) eg. on iOS these things may even be accumulated as "2 messages, 3 reactions".

we can always iterate, maybe let's see if that is really a real-world problem :)

do whatsapp/signal/telegram show a notification for a removed reaction?

Reply:
I use WhatsApp sometimes, but disabled notifications for reactions at all, so can't say much. Also used other messengers before and indeed, i remember that most of them can show notifications for reactions, but don't remember any of them showing notifications for reactions removal. I just thought it's a good idea, but this may be postponed at least